plugins/usbdmx/ShowJockeyFactory.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ShowJockeyFactory.cpp
+ * The WidgetFactory for ShowJockey widgets.
+ * Copyright (C) 2016 Nicolas Bertrand
+ */
+
+#include "plugins/usbdmx/ShowJockeyFactory.h"
+
+#include "libs/usb/LibUsbAdaptor.h"
+#include "ola/Logging.h"
+#include "ola/base/Flags.h"
+
+DECLARE_bool(use_async_libusb);
+
+namespace ola {
+namespace plugin {
+namespace usbdmx {
+
+using ola::usb::LibUsbAdaptor;
+
+const uint16_t ShowJockeyFactory::PRODUCT_ID = 0x57fe;
+const uint16_t ShowJockeyFactory::VENDOR_ID = 0x0483;
+
+bool ShowJockeyFactory::DeviceAdded(
+    WidgetObserver *observer,
+    libusb_device *usb_device,
+    const struct libusb_device_descriptor &descriptor) {
+  if (descriptor.idVendor != VENDOR_ID || descriptor.idProduct != PRODUCT_ID) {
+    return false;
+  }
+
```

Comment:
In their drivers, they only check fo the vendor and product id. So I don't know if it's really relevant to check for the manufacturer and labels.